src/microbe_stage/editor/action_data/EditorCombinableActionData.cs

```diff
@@ -22,3 +22,21 @@ public override CombinableActionData Combine(CombinableActionData other)
 
     protected abstract int CalculateCostInternal();
 }
+
+public abstract class EditorCombinableActionData<TContext> : EditorCombinableActionData
+{
+    /// <summary>
+    ///   What this action was performed on.
```

Comment:
Isn't this comment wrong? Shouldn't this be something like "The optional context this action was performed in. This is additional data in addition to the action target. Not all editors use context info." something like that.